packages/aws-cdk-lib/aws-codecommit/README.md

```diff
@@ -27,6 +27,21 @@ declare const repo: codecommit.Repository;
 repo.notify('arn:aws:sns:*:123456789012:my_topic');
 ```
 
+## Use customer managed key
```

Comment:
Also, considering the priority in this service, I thought it would be better below `## Add initial commit`.